client/service/src/lib.rs

```diff
@@ -92,21 +91,14 @@ impl RpcHandlers {
 	/// would for example send.
 	///
 	/// Returns a `Future` that contains the optional response.
-	///
-	/// If the request subscribes you to events, the `Sender` in the `RpcSession` object is used to
-	/// send back spontaneous events.
-	pub fn rpc_query(
-		&self,
-		mem: &RpcSession,
-		request: &str,
-	) -> Pin<Box<dyn Future<Output = Option<String>> + Send>> {
-		self.0.handle_request(request, mem.metadata.clone()).boxed()
+	//
+	// TODO(niklasad1): support subscriptions?!.
```

Comment:
This looks like a breaking change, but AFAIR it was introduced for Web version of the node in the first place and that's now replaced with Smoldot.

Reply:
yepp, this is hacky the idea is to remove this but I think the Interlay folks is using this feature for running their tests in subxt with an embedded substrate node.

we should double check

Reply:
I'm inclined to remove it, but it'd be good to check with the interlay folks and help them find an alternative.